docs/index.md

```diff
@@ -74,113 +74,16 @@ The following items are limitations with the current release of the Red Hat Clou
-                    "secretsmanager:CreateSecret",
-                    "iam:TagRole",
-                    "secretsmanager:DeleteSecret",
-                    "iam:UpdateOpenIDConnectProviderThumbprint",
-                    "iam:DeletePolicy",
-                    "iam:CreateRole",
-                    "iam:AttachRolePolicy",
-                    "iam:ListInstanceProfilesForRole",
-                    "secretsmanager:GetSecretValue",
-                    "iam:DetachRolePolicy",
-                    "iam:ListAttachedRolePolicies",
-                    "iam:ListPolicyTags",
-                    "iam:ListRolePolicies",
-                    "iam:DeleteOpenIDConnectProvider",
-                    "iam:DeleteInstanceProfile",
-                    "iam:GetRole",
-                    "iam:GetPolicy",
-                    "iam:ListEntitiesForPolicy",
-                    "iam:DeleteRole",
-                    "iam:TagPolicy",
-                    "iam:CreateOpenIDConnectProvider",
-                    "iam:CreatePolicy",
-                    "secretsmanager:GetResourcePolicy",
-                    "iam:ListPolicyVersions",
-                    "iam:UpdateRole",
-                    "iam:GetOpenIDConnectProvider",
-                    "iam:TagOpenIDConnectProvider",
-                    "secretsmanager:TagResource"
-                ],
-            "Resource": [
-                    "arn:aws:secretsmanager:*:<ACCOUNT_ID>:secret:*",
-                    "arn:aws:iam::<ACCOUNT_ID>:instance-profile/*",
-                    "arn:aws:iam::<ACCOUNT_ID>:role/*",
-                    "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/*",
-                    "arn:aws:iam::<ACCOUNT_ID>:policy/*"
-                ]
-            },
-            {
-                "Sid": "VisualEditor1",
-                "Effect": "Allow",
-                "Action": [
-                   "s3:PutObject",
-                   "s3:GetObject",
-                   "s3:AbortMultipartUpload",
-                   "iam:ListRoles",
-                   "s3:ListBucket",
-                   "sts:AssumeRoleWithWebIdentity",
-                   "s3:DeleteObject",
-                   "s3:GetBucketPolicy",
-                   "s3:GetObjectVersion",
-                   "s3:ListMultipartUploadParts",
-                   "s3:PutInventoryConfiguration",
-                   "s3:CreateBucket",
-                   "s3:PutBucketPolicy",
-                   "s3:DeleteBucketPolicy"
-                ],
-                "Resource": "*"
-            }
-        ]
-    }
-    ```
+* You completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements.
+* You generated an offline [OCM token](https://console.redhat.com/openshift/token/rosa). This token is generated through the Red Hat Hybrid Cloud Console. Do not share this token.
+* You have confirmed that you have [GoLang version 1.20 or newer](https://go.dev/doc/install) installed.
```

Comment:
GoLang is only necessary for development of the provider. End-users should not need GoLang at all.